File: AMS.Web/src/app/models/manage-job-role/job-role-response.ts
import { OperationStatus } from "../shared/operation-status";
import { JobRole } from "./job-role";

export class JobRoleResponse 
{
    //#region Constructor

    constructor()
    {
        this.JobRoles = new Array<JobRole>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    JobRoles: Array<JobRole>;

    OperationStatus: OperationStatus;

    //#endregion
}

File: AMS.API/DTO/JobRole/JobRoleResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.JobRole
{
    public class JobRoleResponse
    {
        #region Constructor

        public JobRoleResponse()
        {
            JobRoles = new List<JobRole>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<JobRole> JobRoles { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}